pallets/dapp-staking/src/benchmarking/mod.rs

```diff
@@ -1070,6 +1070,73 @@ mod benchmarks {
         );
     }
 
+    /// Benchmark a single step of mbm migration.
```

Comment:
Doesn't the number of chunks influence the weight of a single step? Why 2 here?

Reply:
Given AccountLedger has high pov (6560), it runs out of block length before it reaches weight_time limit.